Gemfile

```diff
@@ -3,10 +3,12 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'multipart-post', '~> 2.4.1', require: false
+gem 'net-http', '~> 0.6.0', require: false
 
 group :development, :test do
   # ref: http://docs.rubocop.org/en/latest/installation/
   gem 'rack', '~> 3.0'
+  gem 'rbs', '~> 3.9.2'
```

Comment:
if we don't write version explicitly, github actions runner uses 2.9.x and throws error.